Client/src/Features/Auth/authSlice.js

```diff
@@ -27,7 +29,9 @@ export const register = createAsyncThunk(
 
 export const login = createAsyncThunk("auth/login", async (form, thunkApi) => {
   try {
-    const res = await axiosInstance.post(`/auth/login`, form);
+    // Convert email to lowercase before sending the request
+    const modifiedForm = { ...form, email: form.email.toLowerCase() };
```

Comment:
Same with login, this would ideally be done on the login page before data is dispatched to the thunk.

Best to keep all data modification in one place, otherwise we have to look in two places when figuring out how the data is maniuplated as a whole